gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
-#Tue Apr 18 18:43:33 CEST 2017
+#Wed Apr 18 16:00:24 CEST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
+distributionSha256Sum=7a2c66d1a78f811d5f37d14630ad21cec5e77a2a4dc61e787e2257a6341016ce
```

Comment:
What is this line used for?

Reply:
The gradle wrapper downloads the gradle binary (https\://services.gradle.org/distributions/gradle-4.4-all.zip) and can verify if the binary matches the hash sum. It adds a bit more security, not much, but why not.